rllib/algorithms/impala/impala.py

```diff
@@ -209,8 +209,6 @@ def training(
             learner_queue_timeout: Wait for train batches to be available in minibatch
                 buffer queue this many seconds. This may need to be increased e.g. when
                 training with a slow environment.
-            max_requests_in_flight_per_sampler_worker: Level of queuing for sampling
```

Comment:
I know it's a lot of work, but can we actually properly deprecate all these call args?
We should never leave a user with some e.g. yaml file that has these deprecated keys in them and RLlib just explodes on them without them knowing what's going on and how to replace the now "invalid key".

Reply:
nothing is being deprecated here :)
I moved this parameter to the base AlgorithmConfig. So basically, it's available for all algorithms now. although it may not be relevant for the few algorithms that don't. use a WorkerSet.

Reply:
Ah, got it. Sorry, I didn't see that these were added to AlgorithmConfig superclass. Makes perfect sense to move it there!